packages/webdriverio/src/types.ts

```diff
@@ -121,6 +121,11 @@ export interface ChainablePromiseArray extends AsyncIterators<WebdriverIO.Elemen
      * get the `WebdriverIO.Element[]` list
      */
     getElements(): Promise<WebdriverIO.ElementArray>
+
+    /**
+     * Returns an async iterator of key/value pairs for every index in the array.
+     */
+    entries(): AsyncIterableIterator<[number, WebdriverIO.Element]>
```

Comment:
Also this only updates the types, we may have to also enhance `packages/wdio-utils/src/pIteration.ts` with an implementation of this.

Reply:
Should be done